### ISSUE TITLE
Code to process raw dataset to simplified dataset

### ISSUE THREAD

**Issue by michaelhuang74** [googlecreativelab/quickdraw-dataset#33]
The simplification process is discussed as follows. 
1. Align the drawing to the top-left corner, to have minimum values of 0.
2. Uniformly scale the drawing, to have a maximum value of 255.
3. Resample all strokes with a 1 pixel spacing.
4. Simplify all strokes using the Ramer–Douglas–Peucker algorithm with an epsilon value of 2.0.

Where can I find the source code that implements the above steps for simplifying the raw dataset? Thanks.

**Comment by hardmaru** [comment]
I’m not sure if this helps (I don’t have the code to do all 4 steps) but I
ported RDP over to JS a while back, and this is the RDP implementation used
in sketch-rnn-js

https://github.com/tensorflow/magenta-demos/blob/master/sketch-rnn-js/data_tool.js

**Comment by michaelhuang74** [comment]
@hardmaru Thank you very much for sharing your implementation of RDP algorithm.

**Comment by PengBoXiangShang** [comment]
@michaelhuang74   Hello,  have you got the source code that implements the four preprocessing steps? Thanks.

**Comment by michaelhuang74** [comment]
@PengBoXiangShang No, I didn't. It was almost one year ago. I had moved to other things. Sorry.

**Comment by PengBoXiangShang** [comment]
@michaelhuang74   Thanks for your reply.

**Comment by PengBoXiangShang** [comment]
@HalfdanJ   Hello dear Jonas, could you please release the code to covert the raw data to simplified data? Thank you very much.
